### PR TITLE
add response pkg

### PR DESCRIPTION
**Pull request by veezhang**
as title

### CODE REVIEW

**Review by kqzh — DISMISSED** [review]
LGTM

File: response/standard_handler.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+)
+
+func NewStandardHandler(params StandardHandlerParams) Handler {
+	return &standardHandler{
+		params: params,
+	}
+}
+
+func (h *standardHandler) GetStatusBody(r *http.Request, data interface{}, err error) (httpStatus int, body interface{}) {
+	httpStatus = http.StatusOK
```
Comment (nianiaJR) [inline response/standard_handler.go:51]: Why not use GetResponse? StatusBody seems to be like just returning one field

Comment (veezhang) [reply response/standard_handler.go:51]: `StatusBody` means `Status` and `Body`.

`GetResponse` is easily misunderstood as `http.Response`.

**Review by nianiaJR — APPROVED** [review]
LGTM

**Pull request merged**